parcels/particleset.py

```diff
@@ -382,6 +382,69 @@ def show(self, **kwargs):
             print('Plot saved to '+savefile+'.png')
             plt.close()
 
+    # def density(self, field, particle_val=None):
```

Comment:
Ah, apologies- given it's been a while I made and PARCELS commits, I've gotten a bit sloppy. Spent too much time throwing together IKAMOANA code in a rush whilst in Nouméa! You're right, there are old versions and debug statements which I will clean up!